src/lib/logger.js

```diff
@@ -1,4 +1,9 @@
+import set from 'lodash.set'
```

Comment:
should we add a library, just for a small set ?

can we instead just use a simple === ?

this would make build lighter.

Reply:
well cozy-flags is used like that from what I saw, just wanted to stay in the same line

Reply:
cozy-flags does not have lodash.set as dependencies :x: